### ISSUE TITLE
Test for lhapdf info files.

### ISSUE THREAD

**Issue by scarlehoff** [NNPDF/nnpdf#1729]
We have had a few problems with the creation of lhapdf `.info` files. These bugs were flying under the radar for a while since for most of the study we do we don't go fully through `lhapdf` so the fact that were broken didn't affect us.

@cschwan @andreab1997 
could you list here some of the errors / bugs to have an idea of what to test for exactly?

**Comment by cschwan** [comment]
1. the first element of `AlphaS_Qs`, squared, should be equal to `QMin`
2. the number of elements of `AlphaS_Qs` should be equal to the number of elements of `AlphaS_Vals`
3. `AlphaS_OrderQCD` should correspond to `x` in `NxLO`, so `0` for LO, `1` for NLO, ...
4. check that `Particle` is set to an integer in any case!

Conditions 1-3 weren't fulfilled for the QED fit.

**Comment by andreab1997** [comment]
> 1. the first element of `AlphaS_Qs`, squared, should be equal to `QMin`
> 2. the number of elements of `AlphaS_Qs` should be equal to the number of elements of `AlphaS_Vals`
> 3. `AlphaS_OrderQCD` should correspond to `x` in `NxLO`, so `0` for LO, `1` for NLO, ...
> 4. check that `Particle` is set to an integer in any case!
> 
> Conditions 1-3 weren't fulfilled for the QED fit.

Indeed, this is a full list of what already happened. If you want a full list of what could happen in the future I am not sure I can provide one but I can try.

**Comment by RoyStegeman** [comment]
I think the `lhapdf` people are planning to do something similar: https://gitlab.com/hepcedar/lhapdf/-/merge_requests/64 (though we may not want to wait on that). 

Also, I don't think the QED fit bugs would have been found since it was done only locally by Niccolo, right?
